packages/bpk-component-input/src/BpkClearButton.tsx

```diff
@@ -30,14 +31,19 @@ const getClassName = cssModules(STYLES);
 const ClearButtonIcon = withButtonAlignment(ClearIcon);
 
 type Props = {
-  label: string,
-  onClick: (SyntheticInputEvent<HTMLButtonElement>) => mixed,
-  className: ?string,
+  label: string;
+  onClick: (event: SyntheticEvent<HTMLButtonElement>) => void | null;
+  className?: string | null;
+  [rest: string]: any; // Inexact rest. See decisions/inexact-rest.md
 };
 
-const BpkClearButton = (props: Props) => {
+const BpkClearButton = ({
+  className = null,
```

Comment:
I understand it's there for historical reasons (default value for optional prop), but now it should not required, so can be avoided to save some bites? 👀